topbeat/tests/system/test_filesystem.py

```diff
@@ -17,8 +17,8 @@ def test_filesystems(self):
             process_stats=False,
             filesystem_stats=True
         )
-        topbeat = self.start_topbeat()
-        self.wait_until(lambda: self.output_has(lines=1))
+        topbeat = self.start_beat()
+        self.wait_until(lambda: self.log_contains(msg="output worker: publish"))
```

Comment:
Having `output worker: publish` in the log doesn't necessarily guarantee that the data has been flushed to disk. Was there some reason to switch from `output_has(lines=1)`? I think this might be the cause of some of the [build failures](https://ci.appveyor.com/project/elastic-beats/beats/build/848/job/8on6acq7icimv11u#L157) on AppVeyor.